Example/HelloWorld/Package.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version: 5.9
```

Comment:
```suggestion
// swift-tools-version: 6.1
```

Should be 6.1 given the Xcode requirements we have right now.